plugin/db/pg/pg.go

```diff
@@ -1435,7 +1435,7 @@ func getTableConstraints(txn *sql.Tx) (map[string][]*tableConstraint, error) {
 func getViews(txn *sql.Tx) ([]*viewSchema, error) {
 	query := "" +
 		"SELECT table_schema, table_name, view_definition FROM information_schema.views " +
-		"WHERE table_schema NOT IN ('pg_catalog', 'information_schema');"
+		"WHERE table_schema NOT IN ('pg_catalog', 'information_schema') AND view_definition IS NOT NULL;"
```

Comment:
Maybe we use sql.NullString when retrieving the corresponding column and also the relevant github issue as a comment?

Reply:
I think we need to further ask the user. This error happens during creating database, the root cause doesn't seem clear yet.